src/usage.rs

```diff
@@ -0,0 +1,23 @@
+//! The module for the usage advice.
+//!
+//! The usage advice defined in this module is specialised regarding the needs
+//! of this application.  This is,
+//!
+//! * the application name is hard coded,
+//! * the tool tip is hard coded, and
+//! * the expected order of arguments in the usage template is hard coded.
+
+use getopts::Options;
+
+/// A small in-app documentation.
+///
+/// * `options` - The options the render the tool tips for.
```

Comment:
Hmm, it seems rust does not really have a standard for documenting function parameters. (See [here](https://github.com/rust-lang/rust/issues/57525) for progress on that matter)

Since the parameter is already covered by the paragraph below I would remove this. 😅

Reply:
I oriented on the documentation of `getopts :: Options` and the way a good Doxygen documentation is built.

Reply:
The very good [documentation of the std lib](docs.rs/std) uses itself little to non explicit parameter documentation. I think that style would be more important than doxygen. 😅

Reply:
Done.